src/openforms/forms/api/serializers/form.py

```diff
@@ -245,16 +329,79 @@ def get_fields(self):
             fields["payment_backend"].choices = [
                 ("", "")
             ] + payment_register.get_choices()
+
+        # add fields for v2 backwards compatibility
+        if "registration_backends" in fields:
+            fields["registration_backend"] = fields[
+                "registration_backends"
+            ].child.get_fields()["backend"]
+            fields["registration_backend"].required = False
+            fields["registration_backend"].allow_null = True
```

Comment:
this was not originally nullable in v2 - is this deliberate?

Reply:
The admin UI was sending nulls, but I've since fixed that.